qcdb/tests/nwchem_tests/test_tddft_n2+.py

```diff
@@ -35,39 +34,41 @@ def check_tddft(return_value):
+    assert compare_values(root2_ex, qcdb.variable('TDDFT ROOT 2 EXCITATION ENERGY - B3U SYMMETRY'), 5, 'tddft root 2 excitation')
+    assert compare_values(root3_ex, qcdb.variable('TDDFT ROOT 3 EXCITATION ENERGY - B1U SYMMETRY'), 5, 'tddft root 3 excitation')
+    assert compare_values(root4_ex, qcdb.variable('TDDFT ROOT 4 EXCITATION ENERGY - B1U SYMMETRY'), 5, 'tddft root 4 excitation')
+    assert compare_values(root5_ex, qcdb.variable('TDDFT ROOT 5 EXCITATION ENERGY - B1U SYMMETRY'), 5, 'tddft root 5 excitation')
+    assert compare_values(root6_ex, qcdb.variable('TDDFT ROOT 6 EXCITATION ENERGY - AU SYMMETRY'), 5, 'tddft root 6 excitation')
+    assert compare_values(root7_ex, qcdb.variable('TDDFT ROOT 7 EXCITATION ENERGY - AU SYMMETRY'), 5, 'tddft root 7 excitation')
+    assert compare_values(root8_ex, qcdb.variable('TDDFT ROOT 8 EXCITATION ENERGY - AU SYMMETRY'), 5, 'tddft root 8 excitation')
+    assert compare_values(root9_ex, qcdb.variable('TDDFT ROOT 9 EXCITATION ENERGY - AU SYMMETRY'), 5, 'tddft root 9 excitation')
+    assert compare_values(root10_ex, qcdb.variable('TDDFT ROOT 10 EXCITATION ENERGY - B1U SYMMETRY'), 5, 'tdddft root 10 excitation')
+    assert compare_values(root1_energy, qcdb.variable('TDDFT ROOT 1 EXCITED STATE ENERGY - B2U SYMMETRY'), 5, 'tddftroot 1 excited state')
+    assert compare_values(root2_energy, qcdb.variable('TDDFT ROOT 2 EXCITED STATE ENERGY - B3U SYMMETRY'), 5, 'tddft root 2 excited state')
+    assert compare_values(root3_energy, qcdb.variable('TDDFT ROOT 3 EXCITED STATE ENERGY - B1U SYMMETRY'), 5, 'tddft root 3 excited state')
+    assert compare_values(root4_energy, qcdb.variable('TDDFT ROOT 4 EXCITED STATE ENERGY - B1U SYMMETRY'), 5, 'tddft root 4 excited state')
+    assert compare_values(root5_energy, qcdb.variable('TDDFT ROOT 5 EXCITED STATE ENERGY - B1U SYMMETRY'), 5, 'tddft root 5 excited state')
+    assert compare_values(root6_energy, qcdb.variable('TDDFT ROOT 6 EXCITED STATE ENERGY - AU SYMMETRY'), 5, 'tddft root 6 excited state')
+    assert compare_values(root7_energy, qcdb.variable('TDDFT ROOT 7 EXCITED STATE ENERGY - AU SYMMETRY'), 5, 'tddft root 7 excited state')
+    assert compare_values(root8_energy, qcdb.variable('TDDFT ROOT 8 EXCITED STATE ENERGY - AU SYMMETRY'), 5, 'tddft root 8 excited state')
+    assert compare_values(root9_energy, qcdb.variable('TDDFT ROOT 9 EXCITED STATE ENERGY - AU SYMMETRY'), 5, 'tddft root 9 excited state')
+    assert compare_values(root10_energy, qcdb.variable('TDDFT ROOT 10 EXCITED STATE ENERGY - B1U SYMMETRY'), 5, 'tddft root 10 excited state')
 
 @using_nwchem
 def test_1_dft():
     n2_plus = qcdb.set_molecule('''
+        1 2 
         N 0.0 0.0 -0.54885
         N 0.0 0.0  0.54885
+        symmetry d2h
         ''')
 
     qcdb.set_options({
         'basis': '6-31g**',
         'memory' : '3000 mb',
+        'nwchem_charge' : 1,
```

Comment:
this is redundant, right?

Reply:
Yes, I'll remove that in the next set of updates.